snippets/capabilities-table.adoc

```diff
@@ -20,9 +20,7 @@ The following table describes the `baselineCapabilitySet` values.
 |Specify this option when you want to enable the default capabilities for {product-title} 4.13. By specifying `v4.13`, capabilities that are introduced in newer versions of {product-title} are not enabled. The default capabilities in {product-title} 4.13 are `baremetal`, `marketplace`, `openshift-samples`, `Console`, `Insights`, `Storage`, `CSISnapshot` and `NodeTuning`.
 
 |`v4.14`
```

Comment:
Capability MachineAPI is also added into v4.11/v4.12/v4.13 capability set on 4.14 release.
see comments from https://issues.redhat.com/browse/OSDOCS-6810?focusedId=23055385&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-23055385